client_options.go

```diff
@@ -101,18 +101,13 @@ func (w *signRequest) Configure(c *WalletClient) {
 	c.signRequest = w.Sign
 }
 
-// initializeAccessKey handles the specific initialization of the access key.
-func (w *accessKeyConf) initializeAccessKey() (*bec.PrivateKey, error) {
-	var err error
-	var privateKey *bec.PrivateKey
-	var decodedWIF *wif.WIF
-
-	if decodedWIF, err = wif.DecodeWIF(w.AccessKeyString); err != nil {
-		if privateKey, err = bitcoin.PrivateKeyFromString(w.AccessKeyString); err != nil {
-			return nil, errors.Wrap(err, "failed to decode access key")
+func (w *accessKeyConf) initializeAccessKey() (*ec.PrivateKey, error) {
+	privateKey, err := ec.PrivateKeyFromWif(w.AccessKeyString)
+	if err != nil {
+		privateKey, _ = ec.PrivateKeyFromHex(w.AccessKeyString)
```

Comment:
Don't we need the actual error? We could, potentially, wrap it.

Reply:
This func doesn't return error, it returns private key and pub key, and I don't need pub key in this case.